README.md

```diff
@@ -207,6 +207,19 @@ Useful if some specific configuration like --user-principal=host/name@REALM or -
 See man realm for details.
 Example: ad_integration_join_parameters: "--user-principal host/client007@EXAMPLE.COM"
 
+#### ad_integration_sssd_custom_settings
+
+A list of custom setting to be included into the `[domain/$REALM]` section
+of the sssd.conf file. See sssd.conf man pages for details.
+Example:
+
+
```

Comment:
```suggestion
```

markdownlint does not like having two blank lines here - just remove one of the blank lines